examples/external-cluster/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "name" {
```

Comment:
I'm no real fan of adding variables.tf and terraform.tfvars.example to an example directory.

Preferably we just supply a simple/clean example where the module is called with some hardcoded arguments. It makes it more transparent and understandable for a user.

Reply:
Agreed! Changed it